pkg/api/service_cache.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/peterbourgon/fastly-exporter/pkg/filter"
 )
 
+// maxServicePageSize is the maximum amount of results that can be requested
+// from the api.fastly.com/service endpoint.
+const maxServicePageSize = 1000
```

Comment:
Is this number defined or documented anywhere? If you reduce it, will this code start breaking?

Reply:
It **_should_** be documented but it seems like it isn't. I can make sure that's fixed 😄

Reply:
With regards to breakage, it looks like the internal pagination code defaults to using the current maximum instead of returning a 400 or similar if a request provides an out of bounds value. So the exporter will not break if this number gets reduced internally (which I doubt it will).

Reply:
> It **_should_** be documented but it seems like it isn't. I can make sure that's fixed 😄

Sure! Gimme a ping (or PR) when it is, I'd like to update the doc comment here to point to that.